packages/example/remotion.config.ts

```diff
@@ -1,6 +1,6 @@
 import {Config} from 'remotion';
 
-Config.Rendering.setConcurrency(16);
+Config.Rendering.setConcurrency(4);
```

Comment:
We can set it to

```
import os from 'os';
Config.Rendering.setConcurrency(os.cpus().length);
```

So it will work across all our computers

Reply:
okay. that sounds awesome.